spec/factories/ext_management_system.rb

```diff
@@ -89,13 +89,16 @@
   end
 
   factory :ems_openstack, :aliases => ["manageiq/providers/openstack/cloud_manager"], :class => "ManageIQ::Providers::Openstack::CloudManager", :parent => :ems_cloud do
+    after :build do |x|
+      x.authentications << FactoryGirl.create(:authentication, :userid => "admin", :password => "123456789")
+      x.authentications << FactoryGirl.create(:authentication, :userid => "user", :password => "abcdefgh",
+                                              :authtype => "amqp")
```

Comment:
I wonder if it makes sense to keep the authentications separate from the base `ems_openstack` factory.  @Ladas, thoughts?